cl/lib/email_backends.py

```diff
@@ -18,6 +21,93 @@
 )
 
 
+def incr_email_temp_counter(r: Redis) -> None:
```

Comment:
This could use a better name, since it's not just incrementing the temp counter, but also logging the counter when it's full. Perhaps just call it `incr_email_counters`?

Reply:
sure!